spec/fixtures/mauth_signature_testing.json

```diff
@@ -7,12 +7,12 @@
     "verb": "PUT",
     "request_url": "/v1/pictures",
     "body": "<Use the binary value of the blank.jpeg file for the '~_binary' signatures, and an empty string for the '~_empty' signatures>",
-    "query_string": "key=-_.~ !@#$%^*()+{}|:\"'`<>?&∞=v&キ=v&0=v&a=v&a=b&a=c&a=a&k=&k=v"
+    "query_string": "key=-_.~+%21%40%23%24%25%5E%2A%28%29+%7B%7D%7C%3A%22%27%60%3C%3E%3F&%E2%88%9E=v&%E3%82%AD=v&0=v&a=v&a=b&a=c&a=a&k=&k=v"
```

Comment:
Do we need to include other reserved characters as: ` [ ] ; , /` too so that all reserved characters are encoded in the query_string? Because we do not need to encode only these characters ` A-Z, a-z, 0-9, hyphen ( - ), underscore ( _ ), period ( . ), and tilde ( ~ )`.
What do you think ?